portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** A comment on a blog post. */
+public final class Comment {
+
+  private final long id;
+  private final String text;
+  private final long timestamp;
+
+  public Comment(long id, String text, long timestamp) {
```

Comment:
Are you going to be able to leave comments on each page or just the index page?

Reply:
the current implementation will serve the same comments across pages, so there's one comment section for the site. this will change in a future upload, this is just the initial implementation of the feature

Reply:
i've noticed my reply was slightly unclear. in saying that the implementation **will** serve the same comments across pages, i meant that there is only a single store of comments in the current implementation. if the comment feature was implemented on each page, it would always return the same comments. however, the comment section is currently only implemented on the index page. 👍